python-sdk/nuscenes/predict/models/physics.py

```diff
@@ -114,15 +114,15 @@ def _constant_magnitude_accel_and_yaw_rate(kinematics_data: KinematicsData,
     the assumption that the rates of change of speed and yaw are constant.
     :param kinematics_data: KinematicsData for agent.
     :param sec_from_now: How many future seconds to use.
-    :param sampled_at: Timester between predictions (Hz).
+    :param sampled_at: Time step between predictions (Hz).
```

Comment:
@freddyaboulton We should say something like `frequency that predictions are sampled at (Hz)`. Saying time step immediately makes me think of a duration (e.g., sec, ms).